apps/ivrs/api_views.py

```diff
@@ -49,14 +50,20 @@ def get(self, request):
 
         is_invalid = True
 
-        if not is_user_registered(parameters['telephone']):
+	if parameters['telephone'] == None:
+            message = get_message(parameters, is_user_registered=False)
+
+        elif not is_user_registered(parameters['telephone']):
             message = get_message(parameters, is_user_registered=False)
 
         elif not is_data_valid(processed_data):
             message = get_message(parameters, is_data_valid=False)
 
         elif not is_logically_correct(processed_data):
             message = get_message(parameters, is_logically_correct=False)
+        
+        elif not school_id.isdigit():
+           message = get_message(parameters, is_school_id_valid=False)
```

Comment:
The one below checks if the school id exists in the database and that's where it failed previously because that expects integer